lib/behaves.rb

```diff
@@ -31,4 +36,17 @@ def defined_behaviors(klass)
       raise NotImplementedError, "Expected `#{klass}` to define behaviors, but none found."
     end
   end
+
+  def add_injected_behaviours(klass)
```

Comment:
I like the `injected_behaviors` name!

Could you verify if this can inject any kind of behaviors other than just methods? For example:

```ruby
module JobProcessor
  inject do
    include Sidekiq::Worker
  end
  
end
```

Reply:
This works, because `instance_eval` reopens the class and executes this call in the context of the `current_class`.

Reply:
This allows for extending modules too.

Reply:
Awesome, feel free to add the new-found usage into the README if you want, or I could do it later :)

Reply:
I can add this in our read me.

Reply:
I took the liberty to do that, but feel free to add more to it!